src/data.py

```diff
@@ -0,0 +1,48 @@
+import boto3
+import os
+import glob
+import subprocess
+from sklearn.model_selection import train_test_split
+
+def s3_list_subfolders(s3_bucket, s3_path):
+    client = boto3.client('s3')
```

Comment:
Add a doctoring to all the functions please (https://www.python.org/dev/peps/pep-0008/).

Reply:
Done.